roles/backup_file/defaults/main.yml

```diff
@@ -1,2 +1,12 @@
 ---
 # defaults file for backup_file
+backup_file_source: '/etc/hosts'
+backup_file_dest_folder: '/tmp/backups'
+backup_file_dest_suffix: '.bak'
+backup_file_dest_dir_owner: "{{ ansible_user_id }}"
+backup_file_dest_dir_group: "{{ ansible_user_gid }}"
+backup_file_dest_dir_mode: '0750'
+backup_file_dest_owner: "{{ ansible_user_id }}"
+backup_file_dest_group: "{{ ansible_user_gid }}"
+backup_file_dest_mode: '0640'
+backup_file_remote_source: true
```

Comment:
```suggestion
backup_file_remote_source: true

```

Reply:
> 

Interesting note here. This did not show in the linter used in devspaces because of the .yamllint file/config at line 29 has:

`new-line-at-end-of-file: disable`

Whatever linter caught this must have ignored that config.

Reply:
I figured out the issue. File associations weren't set. I added some settings to the .vscode/settings.json to prevent this in the future.